rio/src/RIO/Prelude/Logger.hs

```diff
@@ -8,6 +8,7 @@ module RIO.Prelude.Logger
   , logError
   , logOther
     -- * Running with logging
+  , newLogFunc
```

Comment:
Minor point, but can you add this export _below_ `withLogFunc`, since that's what we want people using in general?